_posts/2020-06-08-solidity-summit-recap.md

```diff
@@ -0,0 +1,108 @@
+---
+layout: post
+published: true
+title: 'Wrapping up the Virtual Solidity Summit 2020'
+date: '2020-06-08'
```

Comment:
Needs to be updated.

Reply:
If date is changed the file should also be renamed.